docs/modules/mssql.md

```diff
@@ -1,14 +1,16 @@
 # [Azure SQL Database](https://azure.microsoft.com/en-us/services/sql-database/)
 
-Open Service Broker for Azure (OSBA) contains three Azure SQL Database services. These services enable you to select the most appropriate provisioning scenario for your needs. These services are:
+Open Service Broker for Azure (OSBA) contains five Azure SQL Database services. These services enable you to select the most appropriate provisioning scenario for your needs. These services are:
```

Comment:
maybe we should change this wording so that we don't need to update it every time we make changes here? Maybe "a number of" would be better.

Reply:
Thanks for the suggestion.